### ISSUE TITLE
Can't set mqttTopic to "actor/{identifier}" in telnet

### ISSUE THREAD

**Issue by byteborg** [xoseperez/espurna#478]
Trying to set mqttTopic to a topic with prefix does not work (nothing happens). Setting it to `{identifier}` works. Am I doing it wrong?

```
set mqttTopic actor/{identifier}
set mqttTopic {identifier}
+OK
set mqttTopic 'actor/{identifier}'
-ERROR unknown command
set mqttTopic "actor/{identifier}"
-ERROR unknown command
```

**Comment by xoseperez** [comment]
Changing any setting related to MQTT topic in the terminal requires a reconnection to the broker. You just have to:

```
set mqttTopic actor/{identifier}
mqtt.reset
```

**Comment by mcspr** [comment]
I think it is #477 
```python
>>> len("set mqttTopic actor/{identifier}\r\n")
34
```

Same blank response when exactly 32, and errors when it moves to the start of the buffer.

**Comment by xoseperez** [comment]
In this case, the fix will be released with 1.12.2
